bluebrain/repo-bluebrain/packages/py-multiscale-run/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMultiscaleRun(PythonPackage):
+    """BBP multiscale simulation orchestrator"""
+
+    homepage = "https://bbpgitlab.epfl.ch/-/ide/project/molsys/multiscale_run"
+    git = "ssh://git@bbpgitlab.epfl.ch/molsys/multiscale_run.git"
+
+    maintainers("tristan0x")
+
+    version("develop", branch="main")
+    version("0.1", tag="0.1")
+
+    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools-scm", type=("build", "run"))
+
+    depends_on("gmsh", type=("build", "run"))
+    depends_on("julia", type=("build", "run"))
+    depends_on("neurodamus-neocortex+ngv+metabolism~~coreneuron", type=("build", "run"))
+    depends_on("py-astrovascpy", type=("build", "run"))
+    depends_on("py-bluepysnap", type=("build", "run"))
+    depends_on("py-diffeqpy@1.1", type=("build", "run"))
+    depends_on("py-jinja2", type=("build", "run"))
+    depends_on("py-julia", type=("build", "run"))
+    depends_on("py-libsonata", type=("build", "run"))
+    depends_on("py-mpi4py", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-pandas", type=("build", "run"))
+    depends_on("py-psutil", type=("build", "run"))
+    depends_on("py-pyyaml", type=("build", "run"))
+    depends_on("py-scipy", type=("build", "run"))
+    depends_on("py-tqdm", type=("build", "run"))
+    depends_on("steps@:5", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("steps@5", type=("build", "run"))
```

With the : spack is allowed to use lower versions if necessary. We cannot go to 4 as there is no intersect there. I also reported the problem here as it occurred today:

https://bbpteam.epfl.ch/project/issues/browse/BBPP40-406

Reply:
Indeed, this was a local test that was unfortunately pushed...
thanks for noticing. I will use `steps@5:` version specifier rather than `steps@5` though.